ruby/pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0-alpha-3</version>
+            <version>3.0.0-beta-3</version>
```

Comment:
This is not the version of the jruby runtime, but the Java version JRuby runtime depends on. Since beta-4 is not  deployed yet, jruby has to depend on an slightly older version of Java runtime to work.